libautoscoper/src/gpu/cuda/RayCaster.cpp

```diff
@@ -43,6 +43,11 @@
 #include <iostream>
 #include <sstream>
 
+#include <cuda.h>
+#include <cutil_inline.h>
+#include <cutil_math.h>
+#include <cutil_create_tex_obj.h>
```

Comment:
Does the order matter ? Or could this be sorted alphabetically ?

Reply:
Yes the order does matter here